Slack.Automation/Promact.Erp.Core/Controllers/TaskReportController.cs

```diff
@@ -69,7 +69,7 @@ public TaskReportController(ITaskMailRepository taskMailReport, IStringConstantR
         public async Task<List<TaskMailUserAc>> TaskMailDetailsReport(string UserId,string UserRole,string UserName)
```

Comment:
This part is belong to @iamronakshah . So he had made changes in his issue #122 .